ui-admin/src/study/StudySettings.tsx

```diff
@@ -143,5 +177,27 @@ export function StudyEnvConfigView({ studyEnvContext, portalContext }:
       {isLoading && <LoadingSpinner/>}
       {!isLoading && 'Save study config'}
     </Button>
+
+    { user?.superuser &&
```

Comment:
Let's either just expose this outright, or maybe make a new permission "study_kit_type_edit", and put it under the "power_user" role.   

Minimizing the amount of stuff we gate behind superuser will minimize the amount of code changes we have to make later